messages/agent.preview.md

```diff
@@ -18,32 +18,38 @@ Before you use this command, you must complete these steps:
 
    a. When specifying the connected app's Callback URL, add this second callback URL on a new line: http://localhost:1717/OauthRedirect
 
-   b. Make note of the user that you specified as the "Run As" user when updating the Client Credentials Flow section.
+   b. When adding the scopes to the connected app, add `Manage user data via Web browsers (web)`.
+
+   c. Make note of the user that you specified as the "Run As" user when updating the Client Credentials Flow section.
 
 2. Add the connected app to your agent as described in the "Add Connected App to Agent" section here: https://developer.salesforce.com/docs/einstein/genai/guide/agent-api-get-started.html#add-connected-app-to-agent.
 
-3. Using the username of the user you specified as the "Run As" user above, authorize your org using the JWT flow, as described in this document: https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_auth_jwt_flow.htm.
+3. Request the specific OAuth scopes needed for the APIs by setting the `SFDX_AUTH_SCOPES` environment variable to `refresh_token sfap_api chatbot_api web api`.
+
+4. Using the username of the user you specified as the "Run As" user above, authorize your org using the web server flow, as described in this document: https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_auth_web_flow.htm
+   IMPORTANT: you must pass `--client-id <CONNECTED-APP-CONSUMER-KEY>` to `sf org login web`, otherwise it will use the default CLI connected app.
+   Press Enter to skip sharing the client secret (`Require Secret for Web Server Flow` in the connected app should be disabled).
```

Comment:
this is asked in step 9 here:
https://developer.salesforce.com/docs/einstein/genai/guide/agent-api-get-started.html#create-a-connected-app